src/components/CartItemsList.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { StyledCartProduct } from "../style";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+
+const CartItemsList = ({ cartItems, handleRemoveFromCart }) => {
+  return (
+    <StyledCartProduct>
+      {cartItems.length === 0 ? (
+        <h2>Your cart is empty.</h2>
+      ) : (
+        <>
+          {cartItems.map((item, index) => (
+            <div className="product-detail-wrapper" key={`${item.id}-${index}`}>
```

Comment:
This is an opportunity to split off into another component. That way you are just mapping the list with a component and the file is easier to understand.